src/index.ts

```diff
@@ -29,23 +29,24 @@ const App = async () => {
     DOM.output().innerHTML = ''
   }
 
-  const showStatus = (text, bg, id = null) => {
+  const showStatus = (text: string, bg?: string, id?: string) => {
     const log = DOM.output()
 
     const line = document.createElement('p')
     line.innerHTML = text
-    line.style.color = bg
+    if (bg) {
```

Comment:
aegir linting should tell you to be checking for explicit boolean like `bg == null` or something here